src/modules/RootPart.rb

```diff
@@ -1240,6 +1240,25 @@ def MountFSTab(fstab, message)
         )
       end
 
+      efivars_path = "/sys/firmware/efi/efivars"
```

Comment:
NP: Maybe it could be defined as a constant?

Reply:
I considered that but it would make the code less readable, as the constant would have to be moved outside the method.